src/components/RadioButton.tsx

```diff
@@ -8,40 +7,35 @@ import Icon from './Icon';
 import * as Expensicons from './Icon/Expensicons';
 import PressableWithFeedback from './Pressable/PressableWithFeedback';
 
-const propTypes = {
+type RadioButtonProps = {
     /** Whether radioButton is checked */
-    isChecked: PropTypes.bool.isRequired,
+    isChecked: boolean;
 
     /** A function that is called when the box/label is pressed */
-    onPress: PropTypes.func.isRequired,
+    onPress?: () => void;
 
     /** Specifies the accessibility label for the radio button */
-    accessibilityLabel: PropTypes.string.isRequired,
+    accessibilityLabel: string;
 
     /** Should the input be styled for errors  */
-    hasError: PropTypes.bool,
+    hasError?: boolean;
 
     /** Should the input be disabled  */
-    disabled: PropTypes.bool,
+    disabled?: boolean;
 };
 
-const defaultProps = {
-    hasError: false,
-    disabled: false,
-};
-
-function RadioButton(props) {
+function RadioButton({isChecked, onPress = () => undefined, accessibilityLabel, disabled = false, hasError = false}: RadioButtonProps) {
```

Comment:
```suggestion
function RadioButton({isChecked, onPress, accessibilityLabel, disabled = false, hasError = false}: RadioButtonProps) {
```

I think we don't need to assign a default value to `onPress`